src/map/status.c

```diff
@@ -13985,6 +13972,27 @@ static void status_check_job_bonus(int idx, const char *name, int class)
 	}
 }
 
+static void status_load_sc_type(void)
+{
+	skill->dbs->db[skill->get_index(SL_ALCHEMIST)].status_type   = (sc_type)MAPID_ALCHEMIST;
+	skill->dbs->db[skill->get_index(SL_MONK)].status_type        = (sc_type)MAPID_MONK;
+	skill->dbs->db[skill->get_index(SL_STAR)].status_type        = (sc_type)MAPID_STAR_GLADIATOR;
+	skill->dbs->db[skill->get_index(SL_SAGE)].status_type        = (sc_type)MAPID_SAGE;
+	skill->dbs->db[skill->get_index(SL_CRUSADER)].status_type    = (sc_type)MAPID_CRUSADER;
+	skill->dbs->db[skill->get_index(SL_SUPERNOVICE)].status_type = (sc_type)MAPID_SUPER_NOVICE;
+	skill->dbs->db[skill->get_index(SL_KNIGHT)].status_type      = (sc_type)MAPID_KNIGHT;
+	skill->dbs->db[skill->get_index(SL_WIZARD)].status_type      = (sc_type)MAPID_WIZARD;
+	skill->dbs->db[skill->get_index(SL_PRIEST)].status_type      = (sc_type)MAPID_PRIEST;
+	skill->dbs->db[skill->get_index(SL_BARDDANCER)].status_type  = (sc_type)MAPID_BARDDANCER;
+	skill->dbs->db[skill->get_index(SL_ROGUE)].status_type       = (sc_type)MAPID_ROGUE;
+	skill->dbs->db[skill->get_index(SL_ASSASIN)].status_type     = (sc_type)MAPID_ASSASSIN;
+	skill->dbs->db[skill->get_index(SL_BLACKSMITH)].status_type  = (sc_type)MAPID_BLACKSMITH;
+	skill->dbs->db[skill->get_index(SL_HUNTER)].status_type      = (sc_type)MAPID_HUNTER;
+	skill->dbs->db[skill->get_index(SL_SOULLINKER)].status_type  = (sc_type)MAPID_SOUL_LINKER;
+	skill->dbs->db[skill->get_index(SL_GUNNER)].status_type      = (sc_type)MAPID_GUNSLINGER;
+	skill->dbs->db[skill->get_index(SL_NINJA)].status_type       = (sc_type)MAPID_NINJA;
```

Comment:
Hmm one more thing,
Those 2 lines are new and not really related to the commit message "reload status type on `@reloadskilldb`", right? they were not part of the original function.

May you split this change into a new commit explaining this part of the change (preferred) or at least mention it on the commit message (not ideal but good enough)?

Just so in future we know why these 2 lines got added and it doesn't look like something went by unnoticed.

Let me know if you need help doing it

Reply:
Oh god, my bad. I accidentally included them from my repo. Will change. Thank you!

Reply:
no worries :) Thank you for fixing it